coderadar-core/src/main/java/io/reflectoring/coderadar/vcs/port/driver/DeleteRepositoryUseCase.java

```diff
@@ -0,0 +1,7 @@
+package io.reflectoring.coderadar.vcs.port.driver;
+
+import java.io.IOException;
+
+public interface DeleteRepositoryUseCase {
```

Comment:
Rename to `DeleteLocalRepositoryUseCase`

Reply:
Deleted